experimental/packages/opentelemetry-api-logs/src/types/Event.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Attributes, AttributeValue } from '@opentelemetry/api';
+import { LogRecord } from './LogRecord';
+
+export abstract class Event implements LogRecord {
+  name: string;
+  domain?: string;
```

Comment:
is it intended that these properties are public and not readonly?

Reply:
I have changed this class to an interface.